.yarnrc

```diff
@@ -0,0 +1 @@
+yarn-path "../oppia_tools/yarn-v1.17.3/bin/yarn"
```

Comment:
Do we have to change this while changing the yarn version? If yes let's add a comment in both place maybe?

Reply:
I don't think that comments are supported in `.yarnrc`, but I added this to `setup.py`